internal/tests/cli/Makefile

```diff
@@ -1,5 +1,36 @@
 all: test
 
+REGISTRY_NAME ?= docker.io/hashicorp
+TEST_VAULT_IMAGE_NAME=vault
+TEST_VAULT_IMAGE ?= $(REGISTRY_NAME)/$(TEST_VAULT_IMAGE_NAME):1.12.2
+TEST_VAULT_PORT ?= 8200
+TEST_VAULT_CONTAINER_NAME ?= boundary-cli-tests-vault
+
+VAULT_ADDR ?= "http://127.0.0.1:$(TEST_VAULT_PORT)"
+VAULT_TOKEN ?= boundarytok
+
+.PHONY: test-vault-up
+test-vault-up:
+	@echo "Using image: $(TEST_VAULT_IMAGE)"
+	@echo " vault addr: $(VAULT_ADDR)"
```

Comment:
Extra space?
```suggestion
	@echo "vault addr: $(VAULT_ADDR)"
```

Reply:
This is purposeful so the echos all align on the `:`.